source/opt/instruction.h

```diff
@@ -217,6 +241,8 @@ class Instruction {
   // Instructions representing OpLine or OpNonLine itself, this field should be
   // empty.
   std::vector<Instruction> dbg_line_insts_;
```

Comment:
Shouldn't these also be an InstructionList?  Though I guess that can be changed later.

Reply:
Yes, they will be.  I wanted to keep the changes smaller.

Reply:
SGTM